src/main/scala/com/horizon/exchangeapi/OrgsRoutes.scala

```diff
@@ -327,6 +340,69 @@ trait OrgsRoutes extends JacksonSupport with AuthenticationSupport {
       }) // end of complete
     }   // end of exchAuth
   }
+  // ====== GET /orgs/{orgid}/status ================================
+  @GET
+  @Path("orgs/{orgid}/status")
+  @Operation(summary = "Returns an org specific status info", description = "Returns the org with the specified id. Can be run by any user in this org or a hub admin.",
+    parameters = Array(
+      new Parameter(name = "orgid", in = ParameterIn.PATH, description = "Organization id.")),
+    responses = Array(
+      new responses.ApiResponse(responseCode = "200", description = "response body",
+        content = Array(new Content(schema = new Schema(implementation = classOf[GetOrgStatusResponse])))),
```

Comment:
Make sure you view the swagger display of this route in your local exchange to confirm it displays the response body correctly. It should in this case, but sometimes we have to explicitly specify it here, instead of it inferring it from the class.